packages/odyssey/src/scss/abstracts/_functions.scss

```diff
@@ -130,3 +130,17 @@ $icons: (
 
   @return url('data:image/svg+xml,#{$encoded}');
 }
+
+$z-index: (
```

Comment:
Dunno where we should store maps, but we can tackle that during our Tokens/Theming work. Adjacent to function is 💯 for now.